content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1058,4 +1144,7 @@ kubectl create clusterrolebinding permissive-binding \
 ```
 {{< /warning >}}
 
+After you have transitioned to use RBAC only, you should adjust the access controls
```

Comment:
the recommendation is to use Node + RBAC authorizers, so saying "RBAC only" is misleading

```suggestion
After you have transitioned to use RBAC, you should adjust the access controls
```